DEVELOPMENT.md

```diff
@@ -1,10 +1,6 @@
 # Development Guide for Athens
 
-The proxy is built on the [Buffalo](https://gobuffalo.io/) framework. We chose
-this framework to make it as straightforward as possible to get your development environment up and running.
-However, **you do not need to install buffalo to run Athens**. 
-
-Buffalo provides nice features like a file watcher for your server, so if you'd like to install Buffalo, download [v0.12.4](https://github.com/gobuffalo/buffalo/releases/tag/v0.12.4) or later to get started on Athens and be sure to download the CLI and put it into your `PATH`.
+The proxy is built on standard library tools. If you know Go, you can read Athens.
```

Comment:
Can you elaborate here. Maybe "The proxy is written in idiomatic Go and uses standard tools. If you know Go, you'll be able to read the code and run the server"?